spec/models/like_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+RSpec.describe Like, type: :model do
+  describe 'associations' do
+    author = User.create(name: 'author', photo: 'photo', bio: 'bio', posts_counter: 2)
```

Comment:
- [ ] I got an error when I tried to run tests, this is because you are using an attribute name that is not available in the schema. Kindly change from posts_counter to posts_count in all the test files and model validations and run the tests before submitting them for another review.

![spec](https://github.com/Thinus01/DailyBlogger/assets/53124548/fa0ff0ce-03d4-413b-8cf2-a6f82a2d4b08)

Reply:
Fixed.